components/Snackbar/src/MDCSnackbarMessage.h

```diff
@@ -187,6 +189,17 @@ extern NSString *__nonnull const MDCSnackbarMessageBoldAttributeName;
 @property(nonatomic, copy, nullable) void (^snackbarMessageWillPresentBlock)
     (MDCSnackbarMessage *_Nonnull message, MDCSnackbarMessageView *_Nonnull messageView);
 
+/**
+ Whether the Snackbar message is transient and automatically dismisses after the provided @c
+ duration time or is not transient and will not dismiss automatically.
+
+ @note: If VoiceOver is turned on, a snackbar will not automatically dismiss if the snackbar has an
+ action, regardless of this property.
+
+ Defaults to YES.
```

Comment:
This seems to be at times in conflict with the duration property. For instance, if a client sets duration to 0.1 it will dismiss unless this property (automaticallyDismiss) is set to NO what would happen? From reading the documentation it sounds like it would dismiss. But what about the scenario where a client sets the duration to 0.0 and the automaticallyDismiss to YES? I'm lost on what property _wins_ if the conflict?

Reply:
I'm unsure where your confusion is.

If there is something you wish I clarify in the documentation of this property I am happy to do so.

Initially I tried to add it to the duration property but it made more sense to add an opt-in property based on feedback. I personally don't think this additional property conflicts with duration. It clearly states it will look into the duration property unless automaticallyDismisses = NO.

Reply:
I'm with Cody on this property being a bit weird and unnecessary, but I understand that you added this property because of prior feedback you received, so 🤷‍♂